packages/commonwealth/client/scripts/state/index.ts

```diff
@@ -180,6 +193,15 @@ const app: IApp = {
     const capacitor = window['Capacitor'];
     return !!(capacitor && capacitor.isNative);
   },
+  platform: () => {
```

Comment:
for the sake of this PR I guess it is fine, however, keep in mind that we are slowly moving away from this file and this state approach as it is not reactive. Eventually, all this logic from this file should land either in zustand or react-query.